src/beerBlock.js

```diff
@@ -13,7 +13,13 @@ class BeerBlock extends React.Component {
         return (
             <div className='framegent'>
                 <h3 className='titleh3'>{this.props.name}</h3>
-                <div className='fakeMap'>MAP IS NOT AVAILABLE</div>
+                <div className='fakeMap'>
+                {this.props.latitude ? <YMaps>
+                    <Map defaultState={{ center: [this.props.latitude, this.props.longitude], zoom: 9,  width:240, height:120 }}>

```

Comment:
i cant check map working due to the following error
Please fix it

![map-error](https://user-images.githubusercontent.com/6079597/133384132-d8f2d8a1-c726-49a6-be8a-d96a9a0d813a.png)